.stylelintrc

```diff
@@ -0,0 +1,48 @@
+{
+  "ignoreFiles": [
+    "**/Public/**",
+    "**/Private/Templates/**",
+    "node_modules/**",
+    "**/*.noLinter.*"
+],
+  "extends": "stylelint-config-standard",
```

Comment:
Wouldn't `stylelint-config-recommended-scss` be enough?
I'm scared that these big config files might intimidate people or confuse them.

Reply:
Good point. I've updated the file. I left some rules inside, as they are for PostCSS rules